Description: Wait a bit for the scheduler to catch up in the lpstat test. Fixes failures on slow architectures like armel.
Bug-Debian: http://bugs.debian.org/670878

Index: cups-1.5.2/test/5.4-lpstat.sh
===================================================================
--- cups-1.5.2.orig/test/5.4-lpstat.sh	2009-04-13 19:03:15.000000000 +0200
+++ cups-1.5.2/test/5.4-lpstat.sh	2012-05-14 10:31:21.364668698 +0200
@@ -15,6 +15,7 @@
 #
 
 echo "LPSTAT Test"
+sleep 10
 echo ""
 echo "    lpstat -t"
 ../systemv/lpstat -t 2>&1
